sdk/fabric/Azure.ResourceManager.Fabric/api/Azure.ResourceManager.Fabric.netstandard2.0.cs

```diff
@@ -0,0 +1,277 @@
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.Fabric.FabricCapacityResource> GetFabricCapacitiesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.Fabric.Models.RpSkuDetailsForNewCapacity> GetSkusFabricCapacities(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.Fabric.Models.RpSkuDetailsForNewCapacity> GetSkusFabricCapacitiesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+}
+namespace Azure.ResourceManager.Fabric.Models
+{
+    public static partial class ArmFabricModelFactory
+    {
+        public static Azure.ResourceManager.Fabric.Models.CheckNameAvailabilityResult CheckNameAvailabilityResult(bool? nameAvailable = default(bool?), Azure.ResourceManager.Fabric.Models.CheckNameAvailabilityReason? reason = default(Azure.ResourceManager.Fabric.Models.CheckNameAvailabilityReason?), string message = null) { throw null; }
+        public static Azure.ResourceManager.Fabric.FabricCapacityData FabricCapacityData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, System.Collections.Generic.IDictionary<string, string> tags = null, Azure.Core.AzureLocation location = default(Azure.Core.AzureLocation), Azure.ResourceManager.Fabric.Models.RpSku sku = null, Azure.ResourceManager.Fabric.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Fabric.Models.ProvisioningState?), Azure.ResourceManager.Fabric.Models.ResourceState? state = default(Azure.ResourceManager.Fabric.Models.ResourceState?), System.Collections.Generic.IEnumerable<string> administrationMembers = null) { throw null; }
+        public static Azure.ResourceManager.Fabric.Models.RpSkuDetailsForExistingCapacity RpSkuDetailsForExistingCapacity(string resourceType = null, Azure.ResourceManager.Fabric.Models.RpSku sku = null) { throw null; }
+        public static Azure.ResourceManager.Fabric.Models.RpSkuDetailsForNewCapacity RpSkuDetailsForNewCapacity(string resourceType = null, string name = null, System.Collections.Generic.IEnumerable<string> locations = null) { throw null; }
+    }
+    public partial class CheckNameAvailabilityContent : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Fabric.Models.CheckNameAvailabilityContent>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Fabric.Models.CheckNameAvailabilityContent>
```

Comment:
Could we rename this model to `FabricCheckNameAvailabilityContent` to avoid name collision between different libraries?